docs/Developers/Git_Basics.rst

```diff
@@ -178,6 +205,17 @@ and your commit will be pushed online.
 Creating a Pull Request
 -----------------------
 
+If you've made it this far, and you're still using the 'master' branch, then
```

Comment:
This is an important request. People need to own their branches, and care for them.